.github/CODEOWNERS

```diff
@@ -0,0 +1,50 @@
+###################################################################################################
+# Overview
+#
+# Pattern used to match files follows most of the same rules as used in gitignore files. Order is
+# important; the last matching pattern takes precendence.
+#
+# For more info see:
+# https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
+###################################################################################################
+
+
+###################################################################################################
+# Global (@keycloak/maintainers)
+###################################################################################################
+
+/**                                                         @keycloak/maintainers
```

Comment:
is `/**` the correct format? Based on [this](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#example-of-a-codeowners-file) they use `*` for referring all directories.

Also, in all the following, to link one directory and all sub-directories, we use `/testsuite/**`, but they use `testsuite/`. I don't have any strong opinion of using one or another, but I am wondering whether it works this way.

Reply:
Both works, but changed it to use the format from the example as that's cleaner